VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -113,13 +125,54 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         );
     }
 
-    getPrivateConsultationParticipants(): ParticipantListItem[] {
+    isJohConsultation(): boolean {
+        return this.roomLabel?.toLowerCase().includes('judgejohconsultationroom') ? true : false;
+    }
+
+    isFirstJohFromGroupMember(participant: any): boolean {
+        if (participant.hearing_role === HearingRole.PANEL_MEMBER)
+            return this.panelMembers[0].id === participant.id;
+        
+        if (participant.hearing_role === HearingRole.WINGER)
+            return this.wingers[0].id === participant.id;
+
+        if (participant.hearing_role === HearingRole.JUDGE)
+            return this.judge.id === participant.id;          
+    }
+
+    isLastJohFromGroupMember(participant: any): boolean {
+        if (participant.hearing_role === HearingRole.PANEL_MEMBER)
+            return this.panelMembers[this.panelMembers.length - 1].id === participant.id;        
+        else if (participant.hearing_role === HearingRole.WINGER)
+            return this.wingers[this.wingers.length - 1].id === participant.id;
+        else if (participant.hearing_role === HearingRole.JUDGE)
+            return this.judge.id === participant.id;   
+        
+        return true;
```

Comment:
Is this the right behaviour? Returning true if the participant is not a Panel Member, Winger or Judge?

Reply:
Function name could be missleading so the name will be updated on next commit.
The function requires to return true if there is a single specific participant-class depending on the:
1 participants group order
2 individual participant